modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/screen/RequestScreen.js

```diff
@@ -257,7 +257,6 @@ class RequestScreen extends Screen {
 				method: httpMethod,
 				mode: 'cors',
 				redirect: 'follow',
-				referrer: 'client',
```

Comment:
Instead of just fully removing `referer`, can we first check on `requestHeaders` to see if referer already exists? And then if it doesn't we use 'client'?


Also, it seems like there might be a distinction between `referrer` as a fetch option, and `request.referer` Is that right? I noticed that the [docs](https://developer.mozilla.org/en-US/docs/Web/API/fetch#referrer) say it should be "same-origin URL, about:client, or an empty string.}

Reply:
> Instead of just fully removing referer, can we first check on requestHeaders to see if `referer` already exists? And then if it doesn't we use 'client'?

Hi @bryceosterhaus, I'm just debugging `requestHeaders` and when we navigate from page A to page B, it doesn't have a `referer`. So if we do that, we will get the same issue. 

> I noticed that the [docs](https://developer.mozilla.org/en-US/docs/Web/API/fetch#referrer) say it should be "same-origin URL, about:client, or an empty string.

If we should pass "same-origin URL, about:client, or an empty string." I think we should use `about:client` because we can let the browser decide on `referrer`. What do you think?

Could you let me know if I understood your questions correctly?

Reply:
If using `about:client` works, lets do that since that is the only one in the documentation that seems to make sense for our case

Reply:
Awesome, it works correctly!! Can you review it again?

Reply:
@bryceosterhaus